Fixed: Make customizations valid

ahankinson opened the pull request:
This PR follows on from #1171 and fixes errors in the customizations.

It should be merged after #1171, since it includes the changes from that PR.

The commits in this PR are separated by the individual changes and the commit message contains a short description on what the problem was.

kepper committed:
add validation for customizations

ahankinson committed:
Fixed: CMN Module was not included in Mensural

No need to remove these classes since they were defined on the CMN
module, and it was not included.

ahankinson committed:
Fixed: Neumes customization

The CMN module was not included, so model classes did not need to be deleted.

Also fixed the name of an attribute class.

ahankinson committed:
Fixed: MEI Basic analytical class

classSpecs are not valid in the "include"; only elementSpec.

ahankinson committed:
Fixed: Elements not in header

The composer, arranger, and lyricist elements are defined in shared, not header.

ahankinson committed:
Fixed: Various misspellings and missing classes

ahankinson committed:
Fixed: classSpec is commented out in the source

these classSpec are commented out, so it was commented out here.

ahankinson committed:
Add Laurent to respStmt and an entry for the dev conference

kepper commented on customizations/mei-basic.xml:
```diff
@@ -11,6 +12,10 @@
+                <respStmt>
+                    <resp>Revisions</resp>
+                    <name xml:id="LP">Laurent Pugin</name>
+                </respStmt>
```
Didn't you just revise it as well? Good catch, but we should be more generous / correct when giving credit…

ahankinson replied:
No, I just fixed the bugs. :-D

ahankinson replied:
The credit is also to "The MEI Community" so I'm covered by that.

kepper reviewed (dismissed):
In general, all fine. I would suggest to also give credit to @ahankinson, though…

bwbohl reviewed (commented):
Still got one validation error in mi-CMN.xml, at least on my machine.
System-ID: /Volumes/VCS/GitHub/bwbohl/music-encoding/customizations/mei-CMN.xml
Haupt-Validierungsdatei: /Volumes/VCS/GitHub/bwbohl/music-encoding/customizations/mei-CMN.xml
Schema: /Volumes/VCS/GitHub/bwbohl/music-encoding/source/validation/mei-customizations.sch
Programmname: ISO Schematron
Fehlerlevel: error
Beschreibung: There is no model class with name "model.scorePart.mensuralAndNeumes".
Anfang: 107:26

ahankinson committed:
Fixed: Comment out missing class

The model.scorePart.mensuralAndNeumes class is commented out in the spec, so it's commented out here.

ahankinson commented:
@bwbohl fixed.

bwbohl commented:
thx @ahankinson

bwbohl commented:
thx @rettinghaus!
